master/buildbot/test/fake/docker.py

```diff
@@ -41,8 +45,9 @@ def build(self, fileobj, tag):
             self._images.append({'RepoTags': [tag + ':latest']})
 
     def create_host_config(self, *args, **kwargs):
-        pass
+        self.call_args_create_host_config.append(kwargs)
 
-    def create_container(self, image, *args, **kwargs):
+    def create_container(self, *args, **kwargs):
```

Comment:
image is still a valid argument. `*args, **kw` catches all possible arguments; therefore, image is still a valid argument. It's just not used in the create_container implementation so there is no need to explicitly catch the argument. I can revert the change if this is a concern.